lodash.src.js

```diff
@@ -6042,6 +6042,14 @@
     }
 
     /*------------------------------------------------------------------------*/
+    var zipWith = restParam(
+        function(foldFunc, lists) {
+          return _.map(unzip(lists), function(list) {
```

Comment:
We can probably use simpler versions like `arrayMap` and friends instead of `_.map`.